src/collections/_documentation/guides/manage-event-stream/index.md

```diff
@@ -0,0 +1,213 @@
+- Build, Run, and Save the following query
+- Notice that you can open an issue details page by clicking on the `issue id` link
+
+    ![Busiest Issues]({% asset guides/manage-event-stream/10.png @path %})
+
+- Alternatively, you can sort the `Issues` table by issue `Frequency`
+
+    ![Issue Frequency]({% asset guides/manage-event-stream/13.png @path %})
+
+### > **How to set proper rate limits?**
+
+A good way to set a project rate limit is by figuring out the expected event volume based on past traffic. Let's look at an example
+
+![Calculating rate limits]({% asset guides/manage-event-stream/14.png @path %})
+
+- Open the project DSN key configuration under `[Project Settings] > Client Keys > [Configure]`
+- Take a look at the `KEY USAGE IN THE LAST 30 DAYS` graph. Max daily rate in the last month is < 326K
+- Based on that, we can define a ceiling **daily** max value of ~330K which is ~13,750 events an **hour**.
+- Notice that you can set a daily, hourly or minute-based rate limit. We'd recommend using an hourly rate to avoid situations where a random event spike might exhaust your daily set quota and leave you blind for the rest of the day.
+- You can always go back, check the graph to see the number of events dropped due to rate limiting and revisit your settings.
+
+    ![Revisit rate limits]({% asset guides/manage-event-stream/15.png @path %})
+
+### > **Spike Protection was activated - what should I do?**
+
+You received an email notifying you that Spike Protection was triggered and applied to your account.
+
+![Spike Protection Email]({% asset guides/manage-event-stream/spike-email.jpg @path %})
+
+Many times an unexpected spike is caused by a new error (or errors) introduced into your code with a new release version. You can search for these errors by:
+
+- Open the Events view from the left side navigation bar.
+- Select `View All Projects` from the Project drop-down to see events from all projects in your org.
+- Find the spike in the events timeline graph and mark the range with your cursor. Notice the events timeline and date filter (on the top filter bar) zooms into the selected date range.
+   ![Event Spike Range]({% asset guides/manage-event-stream/08.png @path %})
+
+- Now, you can use Discover to query the set of events that happened during the spike.
+- Run the query as below to see the busiest issues that contributed to the spike.
+   ![Issues in Spike Range]({% asset guides/manage-event-stream/09.png @path %})
+
+- Open an issue details page by clicking on the `issue id` link, if it's a real issue - assign it to a project team member to resolve it, otherwise - discard it.
+
+Also, consider doing the following:
+
+- Set better rate limits on the DSN keys associated with the spike related projects.
+- If it's a specific release version that's gone bad - add the version Id to the project's Inbound filters to avoid additional errors from that release.
```

Comment:
```suggestion
- If it's a specific release version that has gone bad, add the version Id to the project's Inbound filters to avoid additional errors from that release.
```